<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dictionary || Artemiy Lazarev</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QLabel" name="inputLabel">
        <property name="text">
         <string>Input a word to serach:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="InputLayout">
      <item>
       <widget class="QCheckBox" name="subsequentCheckBox">
        <property name="text">
         <string>Subsequent Character Search</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="resultsLabel">
      <property name="text">
       <string>Results:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="resultEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
